README.md

```diff
@@ -81,6 +81,49 @@ microovn.ovn-sbctl show
 microovn.ovn-appctl -t /var/snap/microovn/common/run/central/ovnnb_db.ctl cluster/status OVN_Northbound
 ```
 
+## Downscaling
+
+Removing MicroOVN cluster member is as easy as running
+
+```shell
+microovn cluster remove <member_name>
+```
+
+MicroOVN will attempt to gracefully remove services running on the removed
```

Comment:
the grammar here is a bit off, because as the removal process is in progress, a member is not yet removed.

What about something like:
MicroOVN will stop and disable any chassis components running on the member (`ovn-controller` and `ovs-vswitchd`), before attempting to gracefully leave any OVN database clusters for members with central components present.

Reply:
This sounds much better. Thank you!